karma.conf.js

```diff
@@ -63,6 +63,42 @@ module.exports = (config) => {
                 pattern: srcOriginalRecursivePath,
                 included: false,
                 served: true
+            },
+            {
+                pattern: 'test/images/*.png',
```

Comment:
Pattern allows us to use a regexp instead of specifying each image type independently.
Please write a RegExp that covers all of types that you want to using with unit tests.

Reply:
I tried to use something like (jpg|gif|png|svg), but it doesn't work. I will try regexp expression.